### PR TITLE
redux-server-rendering

### PR DESCRIPTION
**Pull request by robwise**

### CODE REVIEW

File: app/views/hello_world/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1 class="alert alert-info this-works">Server Rendering</h1>
+<%= react_component("HelloWorldApp",
+                     @hello_world_props,
+                     generator_function: false,
```
Comment (josiasds) [inline app/views/hello_world/index.html.erb:4]: Shouldn't `generator_function` be `true` when using Redux?

File: client/webpack.server.rails.config.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  resolve: {
+    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', 'config.js'],
+    alias: {
+      lib: path.join(process.cwd(), 'app', 'lib'),
+    },
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```
Comment (josiasds) [inline client/webpack.server.rails.config.js:27]: Maybe `NODE_ENV: JSON.stringify(nodeEnv)`?

File: app/views/hello_world/index.html.erb (hunk 1 above)
Comment (justin808) [inline app/views/hello_world/index.html.erb:4]: @josiasds is 100% right, @robwise, @yorzi

Comment (justin808) [inline app/views/hello_world/index.html.erb:4]: I think is already fixed, but not yet merged. I'm tempted to merge and then to generate.

File: client/webpack.server.rails.config.js (hunk 2 above)
Comment (justin808) [inline client/webpack.server.rails.config.js:27]: @robwise @yorzi, I believe @josiasds is right and we have already fixed this. @robwise Please feel free to merge @yorzi's change to master and regenerate.

**Pull request closed without merging**